troposphere/validators.py

```diff
@@ -53,11 +53,11 @@ def integer_list_item_checker(x):
     return integer_list_item_checker
 
 
-def floatingpoint(x):
+def double(x):
```

Comment:
Do we expect `floatingpoint` to be used outside of this package? In this case I will change it of course, although if this is not the case I'd rather clean it up right away since nothing like a `floatingpoint` is found throughout the AWS CloudFormation documentation, right?

Ad 1. This would show a wrong error message stating that a `floatingpoint` is expected.
Ad 2. See first paragraph, if there's external references to it this would probably be the best solution.

Reply:
Do we expect `floatingpoint` to be used outside of this package? In this case I will change it of course, although if this is not the case I'd rather clean it up right away since nothing like a `floatingpoint` is found throughout the AWS CloudFormation documentation, right?

Ad 1. This would show a wrong error message stating that a `floatingpoint` is expected.
Ad 2. See first paragraph, if there's external references to it this would probably be the best solution.

Reply:
Do we expect `floatingpoint` to be used outside of this package? In this case I will change it of course, although if this is not the case I'd rather clean it up right away since nothing like a `floatingpoint` is found throughout the AWS CloudFormation documentation, right?

Ad 1. This would show a wrong error message stating that a `floatingpoint` is expected.
Ad 2. See first paragraph, if there's external references to it this would probably be the best solution.

Reply:
Do we expect `floatingpoint` to be used outside of this package? In this case I will change it of course, although if this is not the case I'd rather clean it up right away since nothing like a `floatingpoint` is found throughout the AWS CloudFormation documentation, right?

Ad 1. This would show a wrong error message stating that a `floatingpoint` is expected.
Ad 2. See first paragraph, if there's external references to it this would probably be the best solution.

Reply:
Good point - it's unlikely it's being used in custom resources, so this is probably fine.  One last thing: can you please merge master into this branch, and just make sure there are no remaining cases?  We've had a few merges, and some may have used floatingpoint.

Thanks - I'll accept/merge this as soon as you have a chance to merge master!